addons/sys_rack/fnc_addRackOnReturn.sqf

```diff
@@ -16,14 +16,18 @@
  */
 #include "script_component.hpp"
 
-params ["_vehicle","_rackId"];
+params ["_vehicle", "_rackId"];
 
-private _rack = _rackId createVehicle [0,0,0];
+private _rack = _rackId createVehicle [-1000,-1000,-1000];
```

Comment:
If you do the PositionAGL thing without converting none of the objects will be found. It was baffling me.